tests/ocall/enc/enc.cpp

```diff
@@ -3,31 +3,28 @@
 
 #include <openenclave/edger8r/enclave.h>
 #include <openenclave/enclave.h>
+#include <openenclave/internal/calls.h>
 #include <openenclave/internal/enclavelibc.h>
 #include <openenclave/internal/fault.h>
 #include <openenclave/internal/globals.h>
 #include <openenclave/internal/sgxtypes.h>
 #include <openenclave/internal/tests.h>
 #include <openenclave/internal/thread.h>
-#include "../args.h"
+#include "ocall_t.h"
 
-OE_ECALL void Test2(void* args_)
+int64_t test2(int64_t in)
 {
-#if 0
-    *((int*)0) = 0;
-#endif
-    Test2Args* args = (Test2Args*)args_;
-    args->out = args->in;
+    return in;
 }
 
-OE_ECALL void Test4(void* args)
+void test4(void)
 {
     unsigned char buf[32];
 
     /* Call into host with enclave memory */
     oe_memset(buf, 0xAA, sizeof(buf));
 
-    if (oe_call_host("Func2", buf) != OE_OK)
+    if (func2() != OE_OK)
```

Comment:
The earlier version passed enclave buffer (buf) to host. 
The new version ought to do that too.